src/components/video-container/Video-container.component.ts

```diff
@@ -41,6 +45,17 @@ export class VideoContainer extends LitElement {
   @connect('castActivated')
   castActivated: string
 
+  @property({ type: String, attribute: 'storage-key' })
+  storageKey: string
+
+  _storageProvider: StorageProvider
+  connectedCallback() {
+    super.connectedCallback();
+    if (this.storageKey) {
+      this._storageProvider = createProvider(this.storageKey)
```

Comment:
I think it might be better to create a StorageController similar to FullscreenController

Reply:
why ? It's just an adapter for browser API

Reply:
I think about cases when someone would like to extend this component - it will be much easier for them to have them own controller